src/components/Wallet/Exchange/ExchangeShowMore/index.js

```diff
@@ -80,7 +80,7 @@ function ExchangeShowMore({
 
         </div>
         <div className={styles.liquidityButtons}>
-          {!isReservedDataEmpty
+          {!isReservedDataEmpty && (lpTokensBalance !== 0 || !liquidity)
```

Comment:
`lpTokensBalance !== 0` probably won't work - `lpTokensBalance` is now an object (BN probably).